share/dictionary/radius/dictionary.arista

```diff
@@ -21,8 +21,9 @@ ATTRIBUTE	WebAuth					6	integer
 ATTRIBUTE	BlockMac				7	string
 ATTRIBUTE	UnblockMac				8	string
 ATTRIBUTE	PortFlap				9	integer
+ATTRIBUTE	Arista-Captive-Portal			10	string
```

Comment:
Hi @hzhongmj ,

We removed the prefix for the attributes in `v4.0.x`, therefore please remove the `Arista-` from here.